.gitignore

```diff
@@ -23,3 +23,5 @@ NashornProfile.txt
 /.gdbinit
 /.lldbinit
 **/core.[0-9]*
+*.rej
```

Comment:
How about use below config instead. It will ignore all the related diff files in root directory and in the any sub-directory.

```
**/*.rej
**/*.orig
```

Reply:
And if you really wanted to just match files in the root directory, you'd have to write `/*.rej`.